lib/tasks/demo_seed.rake

```diff
@@ -326,7 +326,7 @@ namespace :demo  do
           :expires_at         => Time.zone.now+1.year,
           :name_on_card       => 'Paul PI',
           :expiration_month   => '10',
-          :expiration_year    => '2014',
+          :expiration_year    => '2020',
```

Comment:
What if we made it `Date.current.year + 1` or `1.year.from_now.year` so we don't have to revisit this in 5 years.